crt_portal/cts_forms/views.py

```diff
@@ -1179,6 +1176,9 @@ def get(self, request, id=None):
         for key, value in self.get_shared_report_values(requested_query, keys):
             shared_report_fields[key] = value
         shared_report_fields['date_range'] = self.get_report_date_range(requested_query)
+        # Limit the count to 500 here because we have to display all the reports we're batching in a table
+        if requested_query.count() > 500:
+            requested_query = requested_query[:500]
```

Comment:
Running `count` executes a query - could these simply be `requested_query = requested_query[:500]`?

`:500` will take _up to_ 500, but is happy to take less (I think under the hood it adds a `LIMIT 500` to the query)

e.g.:

```
In [1]: from cts_forms import models

In [2]: models.Report.objects.all()[:10]
Out[2]: <QuerySet [<Report: 242148-QTG>, <Report: 281627-PMF>, <Report: 152618-KLW>, <Report: 149278-CKK>, <Report: 134651-ZJL>, <Report: 224647-GGN>, <Report: 134606-VTW>, <Report: 134607-MLQ>, <Report: 178887-CGR>, <Report: 134608-WGD>]>

In [3]: models.Report.objects.filter(pk=242148)[:10]
Out[3]: <QuerySet [<Report: 242148-QTG>]>
```